source/edge-server.txt

```diff
@@ -79,28 +80,26 @@ Edge Server Setup
 
 .. procedure::
 
-   .. step:: Create a Device Sync-Enabled App
+   .. step:: Create an Edge Server Service
 
-      Create an :ref:`App Services App <create-app>` with 
-      :ref:`Device Sync enabled <enable-sync>`. To get started quickly, 
-      :ref:`create a template app <create-template-app>`.
+      You can use Edge Server with MongoDB Drivers and tools in an Edge 
+      Server Service. Or you can use Edge Server with Atlas Device SDK to
+      develop mobile and edge applications with Device Sync, and get the
+      benefits of conflict resolution and roaming between servers.
 
-      Once your App is created, coordinate with your Product or Account 
-      Representative to enable Edge Server.
+      To get started quickly, 
+      :ref:`create a template app <create-template-app>`.
 
    .. step:: Install and Configure the Edge Server
 
       Configure hardware on-site or a cloud server to host your Edge Server.
-      Install the Edge Server on your hardware, complete a basic 
-      ``config.json`` with your configuration details, and install required
-      dependencies.
-
-      Add an authorization provider, and start the Edge Server.
+      Use our command line tool, ``edgectl``, to install and configure an 
```

Comment:
```suggestion
      Use our command line tool ``edgectl`` to install and configure an 
```
[nit] no commas, since I'd consider this a restrictive element